app/views/admin/configs/show.html.erb

```diff
@@ -234,11 +224,59 @@
                                              placeholder: Constants::SiteConfig::DETAILS[:recaptcha_secret_key][:placeholder] %>
                           </div>
                         </div>
+                        <div class="mt-6">
+                          <% if SiteConfig.allow_both_email_signup_and_login %>
+                            <button
+                              class="crayons-btn crayons-btn--danger"
+                              data-action="click->config#activateEmailAuthModal"
+                              <%= SiteConfig.invite_only_mode ? 'disabled' : ''%>>
+                              <% if SiteConfig.invite_only_mode %>
+                                <span
```

Comment:
@lisasy @ludwiczakpawel This spanned `tooltip` is NOT working on the frontend; I believe I have implemented it [as instructed in this post](https://forem.team/picasso/crayons-update-episode-1-39ah). Please help me debug 🙏

Reply:
I think that post has the wrong class in the example, which is why it wasn't working for you! I did some digging and found [this example](https://github.com/forem/forem/blob/125584d3541c627683e84fdad5cb4112d645d33b/app/views/pages/crayons.html.erb#L72) from our codebase, which is the right class :)

Reply:
@msarit hah, yes.. the class name I entered was wrong 🤦‍♂️🤦‍♂️🤦‍♂️ it should be `crayons-tooltip` instead of just `crayons`... Sorry for that mistake!

Thanks @vaidehijoshi for digging this!